app/services/irs_attempts_api/tracker.rb

```diff
@@ -60,6 +60,10 @@ def track_event(event_type, metadata = {})
       end
     end
 
+    def parse_failure_reason(result)
+      return result.to_h[:error_details] || result.errors.presence
+    end
```

Comment:
This worked out well!

(Realistically this only makes sense for form response type objects, but I think it makes enough sense to just keep here, and a name like `parse_failure_reason_from_form_response` might be a bit much.)